sources/packages/workflow/BPMN/camunda-8/assessment-decisions.dmn

```diff
@@ -2371,7 +2371,7 @@
           <text>31868</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_1u665kl">
-          <text>0.0238 / 8 * 12 / 52</text>
+          <text>0.0238 * 3 / 13</text>
```

Comment:
What are those values used for? Shouldn't we use constants? They repeat  a lot...

Reply:
the 0.0238 are the values that are provided to us run the assessment. the " * 3 / 13" is a way to convert that value to the "format" used in our weekly calculations.